app/src/pages/trace/NewSpanAnnotationForm.tsx

```diff
@@ -12,15 +15,30 @@ export type NewSpanAnnotationFormProps = {
 
 export function NewSpanAnnotationForm(props: NewSpanAnnotationFormProps) {
   const { annotationName: name, spanNodeId, onCreated } = props;
+  const { projectId } = useParams();
+  const { timeRange } = useLastNTimeRange();
   const [commit, isCommitting] = useMutation<NewSpanAnnotationFormMutation>(
     graphql`
       mutation NewSpanAnnotationFormMutation(
         $input: CreateSpanAnnotationInput!
         $spanId: GlobalID!
+        $projectId: GlobalID!
+        $annotationName: String!
+        $timeRange: TimeRange!
       ) {
         createSpanAnnotations(input: [$input]) {
           query {
-            node(id: $spanId) {
+            project: node(id: $projectId) {
+              ... on Project {
+                ...ProjectPageHeader_stats
+                ...AnnotationSummaryValueFragment
```

Comment:
refetch _stats to get a potentially net new annotation name (not using a pre-existing one) refetch ...ValueFragment to get updated information when a new annotation is created with a pre-existing name